### PR TITLE
Dilpreet | Stage | UI Implementation of Activity logs

### PR DESCRIPTION
**Pull request by dilpreetsinghofficial**
https://app.clickup.com/t/2adthyp

### CODE REVIEW

File: apps/web-giddh/src/app/activity-logs/activity-logs.component.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                    <td mat-cell *matCellDef="let element">
+                        <span matTooltip="{{ element.user.email }}" matTooltipClass="">{{ element.user.name }}</span>
+                    </td>
+                </ng-container>
+
+                <!-- Date Column -->
+                <ng-container matColumnDef="time">
+                    <th mat-header-cell *matHeaderCellDef>Date</th>
+                    <td mat-cell *matCellDef="let element">
+                        <span matTooltip="{{ element.time2 }}" matTooltipClass=""> {{ element.time }}</span>
```
Comment (the-prateek-mishra) [inline apps/web-giddh/src/app/activity-logs/activity-logs.component.html]: Please check the matTooltip value is set to `element.time2`

File: apps/web-giddh/src/app/activity-logs/activity-logs.component.ts [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    }
+
+    /**
+    * This function will change the page of activity logs
+    *
+    * @param {*} event
+    * @memberof ActivityLogsComponent
+    */
+    public pageChanged(event: any): void {
+        console.log(event);
```
Comment (the-prateek-mishra) [inline apps/web-giddh/src/app/activity-logs/activity-logs.component.ts]: Avoid console.log

File: apps/web-giddh/src/app/activity-logs/activity-logs.component.ts [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+     *
+     * @memberof ActivityLogsComponent
+     */
+    public getActivityLogs(): void {
+        this.isLoading = true;
+        this.activityService.GetActivityLogs(this.activityObj).pipe(takeUntil(this.destroyed$)).subscribe((response) => {
+            setTimeout(() => {
+                if (response && response.status === 'success') {
+                    this.isLoading = false;
+                    console.log(response);
```
Comment (the-prateek-mishra) [inline apps/web-giddh/src/app/activity-logs/activity-logs.component.ts]: Avoid console logs (do search it in complete file and remove it)

File: apps/web-giddh/src/app/services/activity-logs.service.ts [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        return this.http.post(this.config.apiUrl + ACTIVITY_LOGS_API.GET_ACTIVITY_LOGS.replace(':companyUniqueName', encodeURIComponent(this.companyUniqueName)), model).pipe(
+            map((res) => {
+                let data: BaseResponse<any, any> = res;
+                data.request = model;
+                return data;
+            }),
+            catchError((e) => this.errorHandler.HandleCatch<any, any>(e, model)));
+    }
+
+    /**
```
Comment (the-prateek-mishra) [inline apps/web-giddh/src/app/services/activity-logs.service.ts]: Consider removing commented code if unused

**Review by the-prateek-mishra — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/activity-logs/activity-logs.component.html (hunk 1 above)
Comment (dilpreetsinghofficial) [reply apps/web-giddh/src/app/activity-logs/activity-logs.component.html]: Yes sir , we will provide time on hover on date

**Review by the-prateek-mishra — APPROVED** [review]

File: apps/web-giddh/src/app/activity-logs/activity-logs.component.ts [hunk 5]
```diff
@@ -0,0 +1,429 @@
...
+        user: ""
+    };
+
+    constructor(
+        public activityService: ActivityLogsService,
+        public dialog: MatDialog,
+        private generalService: GeneralService,
+        private router: Router,
+        private changeDetection: ChangeDetectorRef,
+        private ActivityLogsService: LogsService,
```
Comment (the-prateek-mishra) [inline apps/web-giddh/src/app/activity-logs/activity-logs.component.ts:106]:
```suggestion
        private activityLogsService: LogsService,
```

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**